Automattic/wp-calypso
# Reader: Add flag for recent feed overhaul

Fixes https://github.com/Automattic/loop/issues/191

## Proposed Changes

* Add flag: `reader/recent-feed-overhaul`
* Add expandable Recent sidebar item and state.

Before | After
---- | ----
![Screen Shot 2024-10-24 at 4 04 30 PM](https://github.com/user-attachments/assets/64ac70f0-2144-4fcb-9042-fd1aebd35613) | ![Screen Shot 2024-10-24 at 4 06 20 PM](https://github.com/user-attachments/assets/735205f9-a1cf-45aa-8457-ea3d6646af50)



## Why are these changes being made?


* pe7F0s-2iv-p2

## Testing Instructions



* Check that there are no changes to the Recent sidebar item on /read.
* Apply the flag: `?flags=reader/recent-feed-overhaul`
* Check that you can expand and close the Recent sidebar item.

## Pre-merge Checklist
